src/Raven.Server/Documents/Replication/Incoming/IncomingReplicationHandler.cs

```diff
@@ -326,19 +318,13 @@ internal class MergedDocumentReplicationCommand : TransactionOperationsMerger.Me
         {
             private readonly long _lastEtag;
             private readonly DataForReplicationCommand _replicationInfo;
-            private readonly ReplicationLatestEtagRequest.ReplicationType _replicationType;
```

Comment:
check the warning in `public LiveReplicationPerformanceCollector.ReplicationPerformanceType GetReplicationPerformanceType()`